styles/style.css

```diff
@@ -1,15 +1,454 @@
-/* reset */
-
-html, body, div, span, iframe,
-h1, h2, h3, h4, h5, h6, p, a, img, q, strong, i, ol, ul, li,
-fieldset, form, label, legend,
-table, caption, tbody, tfoot, thead, tr, th, td,
-article, aside, 
-figure, figcaption, footer, header, nav, section, video {
-	margin: 0;
-	padding: 0;
-	border: 0;
-	font-size: 100%;
-	font: inherit;
-	vertical-align: baseline;
+
+.c-red {
+  color: #f72210;
+}
+
+.bg-red {
+  background-color: #f72210;
+  color: white;
+}
+
+main.container-fluid {
+  max-width: 1330px;
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+/* nav */
+
+#nav-lg {
+  height: 108px;
+  padding: 20px 10px 15px;
+}
+
+.bottom-left div {
+  background-color: rgb(241, 241, 241);
+}
+
+.v-grid div:not(:last-child) {
+  border-bottom: 1px solid rgb(211, 210, 210);
+}
+
+.newsletter > div > div:last-child {
+  border: 1px solid rgb(209, 209, 209);
+}
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please add a id to `div:last-child` element` and delete the left unnecessary rules


<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_